quantum/rgb_matrix_types.h

```diff
@@ -76,11 +68,13 @@ typedef union {
 #define LED_FLAG_UNDERGLOW 0x02
 #define LED_FLAG_KEYLIGHT 0x04
 
+#define ___ 255
```

Comment:
Careful here, some keyboards already define this as `KC_NO` for the `LAYOUT()` macro definition.

Reply:
Hmm, I didn't see any, will double check

Reply:
damn, ya that's bad, fixing it now

Reply:
fixed